NAMESPACE

```diff
@@ -2,8 +2,8 @@
 
```

Comment:
API:

* `register_parser()`
* `select_parsers()` -> `get_parsers()`
  * Defaults to everything
  *  Get rid of `list_parsers()` (it's just `names(get_parsers())`)